app/components/Hero.tsx

```diff
@@ -0,0 +1,43 @@
+// LIBRARY IMPORTS
+import { Box, Heading } from "@chakra-ui/react";
+import Image from "next/image";
+
+// LOCAL IMPORTS
+import heroImage from "../../public/images/hero-photo.png";
+
+// HERO COMPONENT DEFINITION
+export default function Hero() {
+  return (
+    <Box
+      position="relative"
+      textAlign="center"
+      bg="white"
+      minH="calc(38.25vw + 6rem)"
+    >
+      <Image
+        src={heroImage}
+        alt={"machines photo"}
```

Comment:
I believe it's best to not include words like "photo", "image", etc in alt text. [This](https://www.w3.org/WAI/tutorials/images/tips/) is a great resource on tips and tricks for accessibility. They also have an awesome decision tree for alt text which reminds me to review alt text that we're using for logo :)

Reply:
Great resource, I updated the text to be more descriptive and not use 'photo'